pallets/runtime/src/mips.rs

```diff
@@ -263,8 +265,9 @@ decl_module! {
         /// # Arguments
         /// * `proposal` a dispatchable call
         /// * `deposit` minimum deposit value
+        /// * `url` a link to a website for proposal discussion
         #[weight = SimpleDispatchInfo::FixedNormal(5_000_000)]
-        pub fn propose(origin, proposal: Box<T::Proposal>, deposit: BalanceOf<T>) -> DispatchResult {
+        pub fn propose(origin, proposal: Box<T::Proposal>, deposit: BalanceOf<T>, url: Vec<u8>) -> DispatchResult {
```

Comment:
Looks good. Perhaps URL could be optional?

Reply:
Alright, thanks would make the change.